include/SDL3/SDL_gpu.h

```diff
@@ -1751,8 +1757,8 @@ typedef struct SDL_GPURasterizerState
 typedef struct SDL_GPUMultisampleState
 {
     SDL_GPUSampleCount sample_count;  /**< The number of samples to be used in rasterization. */
-    Uint32 sample_mask;               /**< Determines which samples get updated in the render targets. Treated as 0xFFFFFFFF if enable_mask is false. */
-    bool enable_mask;             /**< Enables sample masking. */
+    Uint32 sample_mask;               /**< Ignored, reserved for future use. */
```

Comment:
Likewise.

Reply:
I don't think we need to do anything here, since if `enable_mask` is false (as it will be for a zero-init'd struct) then the `sample_mask` is totally ignored anyway.